### ISSUE TITLE
Monitoring

### ISSUE THREAD

**Issue by abidal3** [ukanth/afwall#975]
Please add monitor functions such as a log of allowed connections, sent and received bytes and etc.

**Comment by Alexzha** [comment]
I agree, a persistant log function would be great.  Right now the logs are pruned quite often, its hard to get a long term view of whats going on...
